### PR TITLE
Switch to ISO 639-3 Language Codes

### LINKED ISSUE #406: Use "terminology" ISO-639 codes
Currently we are using the ISO-639-2 "bibliographic" codes ("ger" for German). It seems to me that these are not terribly widely used and make compatibility with other multilingual resources poorer than they'd be otherwise; i.e., one has to convert, as I did for #393. I would suggest instead that we switch to the more widely used ISO-639-3 codes, which are "terminological" ("deu" for German). This should be relatively straightforward.

Any thoughts @lfashby @ajmalanoski @agutkin @jacksonllee?

### PR DESCRIPTION
**Pull request by jacksonllee**
This pull request switches the ISO 639 language codes to 639-3.

Resolves #406.

- [x] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

### CODE REVIEW

File: wikipron/config.py [hunk 1]
```diff
@@ -105,9 +110,7 @@ def _get_cut_off_date(self, cut_off_date: Optional[str]) -> str:
             logging.info("No cut-off date specified")
             return today.isoformat()
         try:
-            # TODO: when we require Python 3.7+ later, we can do this:
-            #  d = datetime.date.fromisoformat(cut_off_date)
-            d = datetime.datetime.strptime(cut_off_date, "%Y-%m-%d").date()
+            d = datetime.date.fromisoformat(cut_off_date)
```
Comment (kylebgorman) [inline wikipron/config.py:113]: Drive-by, but this is not a great variable name.

**Review by kylebgorman — APPROVED** [review]
Great. Huge improvement all around. One suggestion and one drive-by note but otherwise LGTM.

File: wikipron/config.py [hunk 2]
```diff
@@ -86,16 +86,21 @@ def __init__(
         self.restart_key = None
 
     def _get_language(self, key) -> str:
-        key = key.lower().strip()
-        if key.startswith("proto-"):
+        key = key.strip()
```
[the remark this answers is not in the document]
Comment (jacksonllee) [reply wikipron/config.py:89]: `key = key.lower().strip()` was the original line before this PR. We now have to drop `.lower()` at this early spot for more fine-grained control below. Specifically, this is to allow the user-provided `key` (whatever its case formatting looks like) to have a chance for `iso639.Language.match`'s language code detection.

**Comment by kylebgorman** [comment]
So happy this is done.
